### PR TITLE
ACN_CD_SymantecEndpointProtection_Parser01

### PR DESCRIPTION
**Pull request by chicduong**
## Proposed Changes

  - Symantec Endpoint Protection Parser
  + Sample Data

### CODE REVIEW

File: Parsers/SymantecEndpointProtection/SymantecEndpointProtection.txt [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+//
+// <51>Nov 11 19:43:36 SAMPLE0007 SymantecServer: sam733904,Local Host IP: 192.168.1.113,Local Port: 80,Local Host MAC: CCF9E4A91226,Remote Host IP: 192.168.1.1,Remote Host Name: ,Remote Port: 33424,Remote Host MAC: 2C3AFDA79E71,TCP,Inbound,Begin: 2020-11-11 19:25:21,End Time: 2020-11-11 19:25:28,Occurrences: 4,Application: C:/WINDOWS/system32/NTOSKRNL.EXE,Rule: Block Unapproved Incoming Ports,Location: Default,User Name: sampleuser4,Domain Name: SMPL,Action: Blocked,SHA-256: 5379732000000000000000000000000000000000000000000000000000000000,MD-5: 53797320000000000000000000000000
+//
+//
+Syslog
```
Comment (shainw) [inline Parsers/SymantecEndpointProtection/SymantecEndpointProtection.txt:31]: Missing 'let LogHeader = Syslog'

File: Parsers/SymantecEndpointProtection/SymantecEndpointProtection.txt [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+| extend LocalHostIpAddr = tostring(AgentPacketLogsParser[0]),
+         LocalPortNumber = toint(AgentPacketLogsParser[1]),
+         RemoteHostIpAddr = tostring(AgentPacketLogsParser[2]),
+         RemoteHostName = tostring(AgentPacketLogsParser[3]),
+         RemotePortNumber = toint(AgentPacketLogsParser[4]),
+         Application = tostring(AgentPacketLogsParser[5]),
+         Action = tostring(AgentPacketLogsParser[6]);
+// All Other Logs - Captures all other logs not specifically identified
+let AllOtherLogs = LogHeader
+| where LogType !in ("Agent System Logs","Agent Activity Logs","Agent Behavior Logs", "Agent Traffic Logs","Agent Traffic Logs","Agent Security Logs", "Agent Packet Logs");
```
Comment (shainw) [inline Parsers/SymantecEndpointProtection/SymantecEndpointProtection.txt:170]: "Agent Traffic Logs" is listed twice

**Review by shainw — CHANGES_REQUESTED** [review]
Some minor fixes, also in the sample JSON, there is no LogTypes for AgentSecurityLogs and AgentPacketLogs.  Might be good to include those, although, not required.

File: Parsers/SymantecEndpointProtection/SymantecEndpointProtection.txt (hunk 2 above)
Comment (chicduong) [reply Parsers/SymantecEndpointProtection/SymantecEndpointProtection.txt]: corrections made

File: Parsers/SymantecEndpointProtection/SymantecEndpointProtection.txt (hunk 1 above)
Comment (chicduong) [reply Parsers/SymantecEndpointProtection/SymantecEndpointProtection.txt]: corrections made

**Review by v-jayakal — APPROVED** [review]

**Review by shainw — APPROVED** [review]

**Pull request merged**